webpack/components/AnsibleHostDetail/AnsibleHostDetail.js

```diff
@@ -1,35 +1,38 @@
 import React, { useState } from 'react';
-import { Tabs, Tab, TabTitleText, Label } from '@patternfly/react-core';
-import { InfoCircleIcon } from '@patternfly/react-icons';
+import PropTypes from 'prop-types';
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
 
+import SkeletonLoader from 'foremanReact/components/common/SkeletonLoader';
 import { translate as __ } from 'foremanReact/common/I18n';
 
+import AnsibleVariableOverrides from './components/AnsibleVariableOverrides';
 import './AnsibleHostDetail.scss';
 
-const AnsibleHostDetail = props => {
+const AnsibleHostDetail = ({ response, status }) => {
   // https://projects.theforeman.org/issues/32398
   const [activeTab] = useState('variables');
 
   return (
-    <Tabs activeKey={activeTab} isSecondary>
-      <Tab
-        eventKey="variables"
-        title={<TabTitleText>{__('Variables')}</TabTitleText>}
-      >
-        <div className="host-details-tab-item">
-          <div className="ansible-host-detail">
-            <Label
-              color="blue"
-              icon={<InfoCircleIcon />}
-              style={{ marginTop: '1.5rem' }}
-            >
-              Ansible Variables coming soon!
-            </Label>
+    <SkeletonLoader status={status} skeletonProps={{ count: 5 }}>
+      <Tabs activeKey={activeTab} isSecondary>
+        <Tab
+          eventKey="variables"
+          title={<TabTitleText>{__('Variables')}</TabTitleText>}
+        >
+          <div className="host-details-tab-item">
+            <div className="ansible-host-detail">
```

Comment:
do we need two separate divs for these classes? could we combine them in one or even put the classnames on one of the other components (`Tab` or `AnsibleVariableOverrides`)?

Reply:
I am not a css guru, but I do not think we can combine them since each sets a different bg color. I tried to pull on the inner one into the AnsibleVariableOverrides component, but that results in light gray skeleton on slightly darker gray background which does not look good.